src/api/explorer.js

```diff
@@ -79,6 +80,29 @@ const historyTransferQuery = `
         }
       }
     }
+
+    events(filter: {
```

Comment:
Damn, my bad, the query in ticket was wrong. `events` will be inefficient, as `data.contains` filter is not indexed in DB. Use `soraMinteds` and `soraBurneds` like this: https://github.com/liberland/chain_explorer/pull/17
Sorry!